xExtension-ImageProxy/i18n/en/ext.php

```diff
@@ -5,7 +5,7 @@
 		'proxy_url' => 'Proxy URL',
 		'scheme_http' => 'Proxy HTTP',
 		'scheme_https' => 'Proxy HTTPS',
-		'scheme_default' => 'Proxy unspecified',
+		'scheme_default' => 'Proxy unspecified URL scheme',
```

Comment:
```suggestion
		'scheme_default' => 'Proxy URLs without scheme',
```

Maybe? But as you prefer